base/loading.jl

```diff
@@ -371,6 +425,7 @@ end
 
 # given a directory (implicit env from LOAD_PATH) and a name,
 # check if it is an implicit package
+# TODO: aren't we supposed to first check for the Project file first and see if it declares a path?
```

Comment:
the path entry in `Project` the project file seems to me like a bit of an experiment that turned out to be unused. Pkg doesn't know about it for example.